### PR TITLE
Import gtrnadb json data

### PR DESCRIPTION
**Pull request by blakesweeney**
This adds a parser and importers for all GtRNA data. This is aimed to merge into the improve-ensembl-import branch and not master because I based it off of ensembl import as there were changes that that are useful here. For example, the configuration updates and such. I'm also planning on using some of the work here (notably my changes to how output gets written and the Entry object) in the ensembl branch as I find this implementation easier to follow. 

The programs seem to handle all the data fine with one annoying exception. Sometimes the ENA taxonomy service gives 500's. This tries to deal with that by retrying requests with a backoff and caching data to make as few requests as possible but failures still happens randomly. I think it will be possible to switch to uniprots searches (e.g. http://www.uniprot.org/taxonomy/?sort=&desc=&compress=no&query=id:651822&format=tab). The failures are handled by simply not importing that entry. We can always retry the import until all are processed correctly of course. 

In addition, this does create a directory of secondary structures to import but we have no scripts to load them yet. I can write a pgloader script to do so easily enough if that is the direction we want to go in.

### CODE REVIEW

**Comment by blakesweeney** [comment]
This can now import secondary structure data from GtRNAdb. I've added some simple generic tasks that can import a single file (PGLoadSecondaryStructure) or all structures (PGLoadAllSecondaryStructures). These aren't tied to GtRNAdb task in anyway and have to be run manually once the other parts of the pipeline has been run.

File: luigi/databases/helpers.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+class FailedTaxonId(Exception):
+    """
+    This is raised when we cannot fetch information on a taxon id.
+    """
+    pass
+
+
+@lru_cache()
+def phylogney(taxon_id):
```
Comment (AntonPetrov) [inline luigi/databases/helpers.py]: Please check spelling

**Comment by AntonPetrov** [comment]
> In addition, this does create a directory of secondary structures to import but we have no scripts to load them yet. I can write a pgloader script to do so easily enough if that is the direction we want to go in.

A pgloader script would be very helpful

**Comment by blakesweeney** [comment]
There is a pgloader script in the release-8 branch.

**Pull request merged**